Movies/Application/DetailScreen/View/MovieDetailViewController.swift

```diff
@@ -69,28 +78,43 @@ final class MovieDetailViewController: UIViewController {
         tableview = UITableView(frame: CGRect(x: 0, y: 0, width: tableViewWigth, height: tableViewHeight))
         tableview.dataSource = self
         tableview.delegate = self
-        tableview.register(DetailMovieTableViewCell.self, forCellReuseIdentifier: "DetailCell")
-        tableview.register(CastTableViewCell.self, forCellReuseIdentifier: "CastCell")
+        tableview.register(DetailMovieTableViewCell.self, forCellReuseIdentifier: Constants.detailCellIdentifier)
+        tableview.register(CastTableViewCell.self, forCellReuseIdentifier: Constants.castCellIdentifier)
         tableview.estimatedRowHeight = 200.0
         tableview.rowHeight = UITableView.automaticDimension
         tableview.separatorColor = .clear
         tableview.backgroundColor = .black
         view.addSubview(tableview)
     }
 
-    private func fetchDetailData(url: String) {
-        NetWorkManager.fetchDataDetail(url: url) { [weak self] movieDetail in
-            self?.movieDetail = movieDetail
+    private func addGesture() {
+        let gesture = UITapGestureRecognizer()
+        gesture.addTarget(self, action: #selector(showWebView))
+        view.addGestureRecognizer(gesture)
+    }
+
+    private func updateView() {
+        getDownloadError()
+        viewModel?.updateViewData = { [weak self] in
             DispatchQueue.main.async {
                 self?.tableview.reloadData()
             }
         }
     }
 
-    private func addGesture() {
-        let gesture = UITapGestureRecognizer()
-        gesture.addTarget(self, action: #selector(showWebView))
-        view.addGestureRecognizer(gesture)
+    private func getDownloadError() {
+        viewModel?.showError = { [weak self] in
+            DispatchQueue.main.async {
+                self?.showAlert(
+                    title: "Ошибка загрузки",
```

Comment:
```suggestion
                    title: "Ошибка загрузки",
```
 "Ошибка загрузки" - хардкод